bcdaworker/worker/worker.go

```diff
@@ -164,20 +166,41 @@ func (w *worker) ProcessJob(ctx context.Context, job models.Job, jobArgs models.
 	return nil
 }
 
-func moveFiles(tempDir string, stagingDir string) error {
+func compressFiles(tempDir string, stagingDir string) error {
 	// Open the input file
 	files, err := os.ReadDir(tempDir)
 	if err != nil {
 		err = errors.Wrap(err, "Error reading from the staging directory for files for Job")
 		return err
 	}
+	gzipLevel, err := strconv.Atoi(os.Getenv("COMPRESSION_LEVEL"))
+	if err != nil || gzipLevel < 1 || gzipLevel > 9 { //levels 1-9 supported by BCDA.
+		gzipLevel = gzip.DefaultCompression
```

Comment:
Can we log a warning here if there's an error / the COMPRESSION_LEVEL is misconfigured?